.golangci.toml

```diff
@@ -14,6 +14,12 @@ ignore-generated-header = false
 [linters-settings.misspell]
 ignore-words = ["unknwon"]
 
+[linters-settings.depguard.rules.main]
+allow = [] # allow all
+deny = [
+  { pkg = "io/ioutil", desc = "Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details." },
+]
```

Comment:
It looks like `depguard` complains if we have no rules, by default it will only allow imports from the Go standard library. We have to specify at least one deny rule and an empty allow list to allow all imports, like we do in Grafana:
https://github.com/grafana/grafana/blob/48a1dae834b9567f33ae6b7b2bc1e74b741bb868/.golangci.toml#L16-L24